### PR TITLE
At garnett interactive elements

### PR DESCRIPTION
**Pull request by akemitakagi**
## What does this change?
N/A

## What is the value of this and can you measure success?
N/A

## Does this affect other platforms - Amp, Apps, etc?
N/A

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
N/A

## Screenshots
N/A

## Tested in CODE?
N/A

### CODE REVIEW

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 1]
```diff
@@ -1273,3 +1273,16 @@ $quote-mark: 35px;
         min-height: 180px;
     }
 }
+// *************** Interactive elements ***************
+
+.interactive.element--showcase,
+.interactive.element--supporting {
+    background: $garnett-neutral-5;
+    padding-top: $gs-baseline/4;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/content-garnett/_types.scss]: Is it necessary even without a border? Otherwise can it be moved into the `mq` below?

**Review by regiskuckaertz — APPROVED** [review]
Small question but otherwise 👍

File: static/src/stylesheets/module/content-garnett/_types.scss (hunk 1 above)
Comment (akemitakagi) [reply static/src/stylesheets/module/content-garnett/_types.scss]: @regiskuckaertz You are right, it doesn't need border 0. I have removed them.

**Comment by regiskuckaertz** [comment]
👍

**Pull request merged**